build/broccoli/optimize.js

```diff
@@ -12,17 +10,22 @@ module.exports = function(jsTree) {
   let isProduction = process.env.EMBER_ENV === 'production';
 
   let plugins = [
-    [debugMacros, {
-      envFlags: {
-        source: '@glimmer/env',
-        flags: {
-          DEBUG: !isProduction,
-          CI: !!process.env.CI
-        }
-      },
+    [require.resolve('babel-plugin-debug-macros'), {
```

Comment:
Were the changes in this file needed for Ember compatibility, or just general cleanup?

Reply:
Something happened as I was upgrading that caused the `babel-plugin-debug-macros options have changed` deprecation message to start popping up everywhere, so I figured it would be good to clean it up and update.